pages/newPage.go

```diff
@@ -68,6 +70,17 @@ func NewPageHandler(res http.ResponseWriter, req *http.Request, ps httprouter.Pa
 		categorySlug, pageSlug, err := databaseActions.CreatePage(userInfo.UserID, title, description, address, website, int(category))
 		if err == nil {
 			log.Printf("Created %s!\n", title)
+
+			// Update algolia index
+			page, err := databaseActions.GetPage(categorySlug, pageSlug)
+			if err != nil {
+				return
+			}
+			err = algoliaUtil.ExportPageRecord(page)
+		  if err != nil {
```

Comment:
This line needs to be formatted.

In addition, it would be better as:
if err := algoliaUtil.ExportPageRecord(page); err != nil {
...

That way you don't have to worry about whether or not err needs to be declared.